pyhoma/client.py

```diff
@@ -235,13 +276,18 @@ async def nexity_login(self) -> str:
     async def get_devices(self, refresh: bool = False) -> list[Device]:
         """
         List devices
+        Per-session rate-limit : 1 calls per 1d period for this particular operation (bulk-load)
         """
         if self.devices and not refresh:
             return self.devices
 
         response = await self.__get("setup/devices")
         devices = [Device(**d) for d in humps.decamelize(response)]
+
+        # Cache response
         self.devices = devices
+        if self.setup:
+            self.setup.devices = devices
```

Comment:
So basically here, you don’t cache the devices, but sync with  the /setup entry point?

Reply:
What do you mean? What we do here is overwrite the /setup response with the latest devices response.

Reply:
OK it’s what I can sync :D

By the way, do we use this feature (cache) somewhere?

Reply:
@tetienne not really I think, thus I was also thinking to remove it... However, it doesn't have a lot of extra code and can help other people to make sure they don't call expensive endpoints all the time. :-)